demo/color_picker/requirements.txt

```diff
@@ -0,0 +1 @@
+Pillow
```

Comment:
Confirmed the color picker demos is properly built on the website demos

![Screen Shot 2022-07-12 at 12 40 32 PM](https://user-images.githubusercontent.com/41651716/178547130-1c47e9bc-3ffc-447a-ba48-a8f3a2a85a64.png)

Reply:
Why do we need this requirements file, when `pillow` is a dependency of `gradio`?

Reply:
Why do we need this requirements file, when `pillow` is a dependency of `gradio`?

Reply:
in case gradio changes its dependencies.